game/main.py

```diff
@@ -55,7 +53,12 @@ def mainGame(screen, screen_width, screen_height):
     all_sprites.add(scoreSprite)
 
     def main_game_loop():
+        global running
```

Comment:
Do these need to be global just yet? maybe just make the global only if it's necessary. otherwise LGTM

Reply:
Yeah I think it's a bit messy but tried to get it working other ways and couldn't so stuck with this for now, I might just merge then we can refactor further down the line